ui/app/templates/components/console/log-command.hbs

```diff
@@ -1,4 +1,4 @@
 <div class="is-flex-center">
   <Icon @name="chevron-right" />
-  <pre class="console-ui-command is-font-mono">{{@content}}</pre>
```

Comment:
claire just added this in [this pr](https://github.com/hashicorp/vault/pull/20064/files). Guessing it's okay now with your changes, but just confirming. 
![image](https://user-images.githubusercontent.com/6618863/232159930-ee087c89-c782-4bd1-8af5-94d24b9e4423.png)

Reply:
Yes I noticed that and removed it since the _SFMono-Regular_ font is applied so it must be inherited from higher up.